unpick-definitions/unpick.json

```diff
@@ -1,5 +1,4 @@
 {
-  "version": 1,
-  "unpickGroup": "net.fabricmc.unpick",
-  "unpickVersion": "${version}"
-}
\ No newline at end of file
+  "version": 2,
+  "namespace": "named"
```

Comment:
Just noticed this should have the constants maven def defined here

Reply:
Ive added it back, it seems to work just fine. Just seen its tottally empty, lets merge this with it, and I can clean it up in a follow up PR to fully remove it (e.g stop building+publishing it)